manifests/g/GeoGebra/GeoGebra5/5.0.672.0/GeoGebra.GeoGebra5.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GeoGebra.GeoGebra.5

```

Comment:
```suggestion
PackageIdentifier: GeoGebra.GeoGebraClassic.5
```
Oops, realized that there's a folder `GeoGebraClassic` in `manifests\g\GeoGebra`

Please make sure to move the package to `manifests\g\GeoGebra\GeoGebraClassic\5\5.0.672.0`

Reply:
Wouldn't it make more sense to have it as a own package and not a subversion? I mean it's almost a different program..

Reply:
Also, why is version 6.0.xx in the GeoGebra Classic folder? I tought only 5.xx is classic..

Reply:
from https://www.geogebra.org/download - I see this:
![image](https://user-images.githubusercontent.com/11600822/140305824-be6aeefe-66f8-4726-b4be-a9a9a544f0e8.png)

We have done the same with Geekbench and other programs:
![image](https://user-images.githubusercontent.com/11600822/140305927-4d8e3b99-517b-456d-85b4-f07f7caad5e1.png)

Which means if people wanted to install version 5 they can do `winget install GeoGebra.GeoGebraClassic.5`, and if someone would like to install version 6, they can do `winget install GeoGebra.GeoGebraClassic`.

This allows people to choose which **major** version they would like to install rather than having to fiddle around with the `--version` parameter with the `winget install` command.